cmd/genji/shell/command.go

```diff
@@ -136,3 +142,193 @@ func displaySuggestions(in string) error {
 	fmt.Println()
 	return nil
 }
+
+// dumpTable displays the content of the given table as SQL statements.
+func dumpTable(tx *genji.Tx, t *database.Table) error {
+	var buf bytes.Buffer
+
+	c := fmt.Sprintf("CREATE TABLE %s", t.Name())
+	buf.WriteString(c)
```

Comment:
```suggestion
	c := fmt.Fprintf(&buf, "CREATE TABLE %s", t.Name())
```

Reply:
I should use `buf` because the output can be of two kinds:
```SQL
CREATE TABLE test;

# OR

CREATE TABLE test ( a INTEGER);
```
The `fmt.Fprintf` should be done after the check of `fieldsContraints`.